app/[locale]/login/page.tsx

```diff
@@ -75,7 +75,7 @@ const LoginPage: React.FC = () => {
     };
 
     useEffect(() => {
-        if (errors) {
+        if (errors.length > 0) {
```

Comment:
Needed to update, because otherwise the login page kept scrolling to content on load.